### DISCUSSION TITLE
Laravel Valet: auto reload browser when content changed

### DISCUSSION THREAD

**Discussion by risangbaskoro** [laravel/framework#41127] [Ideas] [answered]
If possible, it would be really nice if Valet can reload the browser automatically when something in a parked directory is changed.

Case: if I have a project called "project" and I am currently opening `http://project.test` in my browser, the browser would automatically reload when I change and saved something in the `project` directory.

**Comment by ralphjsmit** [comment, marked as answer]
Hey, you can use Browsersync for that with Laravel Mix. Here’s an article (I wrote it): [https://ralphjsmit.com/laravel-valet-browsersync/](https://ralphjsmit.com/laravel-valet-browsersync/).
